.prettierrc.yml

```
@@ -1,6 +1,6 @@
 printWidth: 120
 tabWidth: 4
 overrides:
-  - files: "*.{md,yml,yaml,html,css,scss,js,cff}"
+  - files: "**/*.{md,yaml,yml,html,css,scss,js,cff}"
```

Can we order this list?
```suggestion
  - files: "**/*.{cff,css,html,js,md,scss,yaml,yml}"
```